app/views/projects/_form.html.erb

```diff
@@ -6,7 +6,7 @@
         <div class="">
             <%= f.label :project_code %>
             <%= 
-              f.text_field :project_code, :class => 'big-text-field form-control' , :required => true , :readonly => false
+              f.text_field :project_code, :class => 'big-text-field form-control' , :required => true , :readonly => false, :disabled => true
```

Comment:
The form doesn't submit because the input here is disabled and its value is not sent to the server.
Don't forget to test before you push.